src/.github/workflows/pre-commit.yml.jinja

```diff
@@ -15,6 +15,16 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
+{%- if odoo_version < 11 %}
+        with:
+          python-version: "2.7"
+{%- elif odoo_version < 13 %}
+        with:
+          python-version: "3.6"
+{%- elif odoo_version < 14 %}
+        with:
+          python-version: "3.8"
+{%- endif %}
```

Comment:
that is not exactly the same mapping as here : 
https://github.com/OCA/oca-addons-repo-template/blob/master/src/.github/workflows/%7B%25%20if%20ci%20%3D%3D%20'GitHub'%20%25%7Dtest.yml%7B%25%20endif%20%25%7D.jinja#L15
10 -> 2.7
11 -> 3.5
12, 13, 14 -> 3.6
15 -> 3.8
Is it normal ? 
(naive question. Sorry if it's not relevant.)

Reply:
Yeah, it's a bit complicated. I'm not sure what's the best approach...
In the end, maybe we should pin the python version in the pre-commit config itself...
But that would be annoying for people who, say, develop with python 3.10 and don't even have older python versions on their system...

Reply:
Pinning the Python version in pre-commit makes a lot of sense to me. I think following the mapping referenced by @legalsylvain makes the most sense.

Reply:
That's fine with me too and probably the most reliable.